### DISCUSSION TITLE
Number of points in model

### DISCUSSION THREAD

**Discussion by alanrock24** [mworion/MountWizzard4#264] [Q&A] [answered]
Sorry if this is a stupid question, but I'm brand new to MW4.
When I'm on the Model Points page, if I am going to use the Golden Spiral option, is there a way to specify a number of points--lets just say 65 for example, instead of using the presets "Minimal", "Norm", "Medium", or "Maximal"?
Thanks,
Alan

**Comment by mworion** [comment, marked as answer]
Hi Alan,
no there is no exact number to be given. The main reason is that if you filter for a horizon mask anyway I do not know in advance how much that (and other filters) might take away.
Michel
